hack/build-index-image.sh

```diff
@@ -35,22 +34,64 @@ function create_index_image() {
   BUNDLE_IMAGE_NAME="${IMAGE_REGISTRY}/${REGISTRY_NAMESPACE}/${BUNDLE_REGISTRY_IMAGE_NAME}:${CURRENT_VERSION}"
   INDEX_IMAGE_NAME="${IMAGE_REGISTRY}/${REGISTRY_NAMESPACE}/${INDEX_REGISTRY_IMAGE_NAME}:${CURRENT_VERSION}"
 
-  INDEX_IMAGE_PARAM=
-  if [[ -n ${2} ]]; then
-    PREV_INDEX_IMAGE="${IMAGE_REGISTRY}/${REGISTRY_NAMESPACE}/${INDEX_REGISTRY_IMAGE_NAME}:${PREV_VERSION}"
-    INDEX_IMAGE_PARAM=--from-index="${PREV_INDEX_IMAGE}"
-  fi
-
   podman build -t "${BUNDLE_IMAGE_NAME}" -f bundle.Dockerfile --build-arg "VERSION=${CURRENT_VERSION}" .
   podman push "${BUNDLE_IMAGE_NAME}"
 
   # Referencing the unstable bundle digest in the index image, rather than the floating tag, to avoid
   # unalignment between cached index image and fetched bundle image.
   BUNDLE_IMAGE_NAME=$("${PROJECT_ROOT}/tools/digester/digester" --image "${BUNDLE_IMAGE_NAME}")
 
-  # shellcheck disable=SC2086
-  ${OPM} index add --bundles "${BUNDLE_IMAGE_NAME}" ${INDEX_IMAGE_PARAM} --tag "${INDEX_IMAGE_NAME}" -u podman --mode semver
+
+  # File-Based Catalog handling
+  ${OPM} migrate ${INDEX_IMAGE_NAME} fbc-catalog || true
+  if [ ! -d fbc-catalog ]
+  then
+    # The index image is already in file-based format. Extracting its catalog file
+    oc image extract ${INDEX_IMAGE_NAME} --file /configs/catalog.json
+  else
+    # The migration took place
+    mv fbc-catalog/community-kubevirt-hyperconverged/catalog.json catalog.json
+  fi
+
+  ${OPM} render ${BUNDLE_IMAGE_NAME} > bundle.json
+  CSV_NAME=$(jq .name bundle.json | tr -d '"')
```

Comment:
you can use the `-r` flag instead of `tr -d`:

```suggestion
  CSV_NAME=$(jq -r .name bundle.json)
```